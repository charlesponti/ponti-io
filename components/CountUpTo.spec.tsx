import { render } from '@testing-library/react';
import { describe, expect, test } from 'vitest';
import CountUpTo from './CountUpTo';

describe('CountUpTo', () => {
  test('should render render', async () => {
    const { container } = render(<CountUpTo value={100} />);
    expect(container).toMatchSnapshot();
  });
});
