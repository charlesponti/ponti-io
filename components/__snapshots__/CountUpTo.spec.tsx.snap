// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CountUpTo > should render render 1`] = `
<div>
  <span>
    0
  </span>
</div>
`;
